### PR TITLE
Add declarations for stoppable 1.1

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/hunterloftis/stoppable
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/stoppable/stoppable-tests.ts [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import * as http from 'http';
 import stoppable = require('stoppable');
 
-const server: http.Server = stoppable(http.createServer());
-server.close();
+const server = stoppable(http.createServer());
```
Comment (EricByers) [inline types/stoppable/stoppable-tests.ts]: nit: Might be nice to type this to StoppableServer?

File: types/stoppable/stoppable-tests.ts [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+server.stop();
```
Comment (EricByers) [inline types/stoppable/stoppable-tests.ts]: nit: Might be nice to call a function on http.Server so that it shows it is properly extended..

**Review by EricByers — CHANGES_REQUESTED** [review]
Couple of small nits.

File: types/stoppable/stoppable-tests.ts (hunk 1 above)
Comment (jplusje) [reply types/stoppable/stoppable-tests.ts]: Good suggestion, changes have been made.

File: types/stoppable/stoppable-tests.ts (hunk 2 above)
Comment (jplusje) [reply types/stoppable/stoppable-tests.ts]: Good suggestion, changes have been made.

**Review by EricByers — APPROVED** [review]
Looks good, thanks for correctly extending this in the definition!

**Pull request merged**